docusaurus.config.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
     'Documentation for SpiceDB, the open-source standard for building authZ services, and AuthZed.',
   favicon: 'img/favicon.svg',
   url: 'https://authzed.com',
-  baseUrl: env('DOCUSAURUS_BASE_URL', '/'),
+  // Oct 1,2023 dsieczko: baseURL is /docs
+  baseUrl: env('DOCUSAURUS_BASE_URL', '/docs/'),
```

Comment:
The second param is a fallback. Using `/` allows the docusaurus project to be run locally but doesn't affect the prod instance.

Reply:
Right, I figured that out - but using docs as the fallback doesn't change anything locally right? Just mimics the prod environment

Reply:
Also I believe /docs/  is not working as intended because the relative links in production docs rn don't have /docs/ prepended

Reply:
For documentation purposes, the base path not being in links was a separate issue with incorrectly using path in markdown links.

Changing the default value to /docs is fine since we're not running the docs site at docs.authzed.com anymore.